lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -91,8 +100,9 @@ export default function FilePickerApp({
   const showGooglePicker = async () => {
     try {
       setLoadingIndicatorVisible(true);
-      const { id, url } = await googlePicker.showPicker();
-      await googlePicker.enablePublicViewing(id);
+      const picker = /** @type {GooglePickerClient} */ (googlePicker);
+      const { id, url } = await picker.showPicker();
+      await picker.enablePublicViewing(id);
```

Comment:
The `showGooglePicker` function is called when a button is clicked and that button is only shown if `googlePicker` was successfully initialized (ie. is non-null). That's beyond TS's powers of inference though.

Reply:
What you've said makes total sense but I'm having trouble connecting it to this line of code :)

Reply:
Ah, I was referring to this line:

```
const picker = /** @type {GooglePickerClient} */ (googlePicker);
```